CMakeLists.txt

```diff
@@ -135,21 +144,6 @@ find_package( Boost COMPONENTS
   REQUIRED 
```

Comment:
I would have thought that the pycompwa project would not need boost, root, etc directly, but the CMake file of ComPWA will require that. I guess these are modifications for the future

Reply:
Since pycompwa builds the entire ComPWA is requires `boost`, `cmake` etc.

Reply:
I removed those requriements since they are automatically passed to target `ui` via the other ComPWA targets. Anyway in order to build pycompwa, `boost`, `cmake`, and `root` are required.